book.rb

```diff
@@ -0,0 +1,11 @@
+# Book class
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] Kindly set the getter and setter for `:rentals` here instead of getter only.

Reply:
This issue is well resolved! ✔️